tests/databases/api_tests.py

```diff
@@ -758,6 +768,38 @@ def test_test_connection_unsafe_uri(self):
         }
         self.assertEqual(response, expected_response)
 
+    @pytest.fixture()
+    def load_unicode_dashboard(self):
+        data = [
+            {"phrase": "Под"},
+            {"phrase": "řšž"},
+            {"phrase": "視野無限廣"},
+            {"phrase": "微風"},
+            {"phrase": "中国智造"},
+            {"phrase": "æøå"},
+            {"phrase": "ëœéè"},
+            {"phrase": "いろはにほ"},
+        ]
+        tbl_name = "unicode_test"
+
+        # generate date/numeric data
+        df = pd.DataFrame.from_dict(data)
+
+        with self.create_app().app_context():
+            database = get_example_database()
+            schema = {
+                "phrase": String(500),
+                "dttm": Date(),
+                "value": Float(),
+            }
+            obj = create_table_for_dashboard(df, tbl_name, database, schema)
+            obj.fetch_metadata()
+
+            db.session.commit()
+            position = "{}"
+            create_dashboard("unicode-test", "Unicode Test", position, None)
```

Comment:
What's being done on other pytest fixtures is to `yield create_dashboard("unicode-test", "Unicode Test", position, None)` and then clean everything up (on this case delete the dashboard and then delete the table for dashboard)

Reply:
Thanks for the hint. I changed it this way. Thanks to that I also removed deleting dashboard from `conftest.py`